modules/client/test/helpers/deploy-erc1155.ts

```diff
@@ -0,0 +1,19 @@
+
```

Comment:
i don't understand why we are hadcoding the bytecode!?
we could do:
```
const ERC1155Factory = new ContractFactory(
        ERC1155_JSON.abi,
        ERC1155_JSON.bytecode,
        signer
    );
```
or i am missing somthing